tests/installation/bootloader_uefi.pm

```diff
@@ -34,6 +34,12 @@ sub run() {
         send_key "ret";
     }
 
+    # Skip to load bootloader in test of online migration on aarch64
+    # Handle aarch64 image boot by wait_boot called in setup_online_migration
+    if (get_var('ONLINE_MIGRATION') && check_var('ARCH', 'aarch64')) {
+        return;
+    }
```

Comment:
I am not quite sure this will work. Sorry, it looks like a wild guess that this might help if you could not test it. Do you have any aarch64 machine available that you could set up for openQA testing on your site?
Anyway, the impact will be low because it will be executed only in paths which have not been covered so far (online_migration && arch64) so I guess the risk is low. So I will merge it.

*But* in our [extra_tests_in_textmode](https://openqa.suse.de/tests/1007684) we use "boot_to_desktop" which uses wait_boot internally to reach the already installed system. I assume it would be a better choice to schedule that module instead of any module from within "installation/bootloader…". IMHO the bootloader modules within installation already do too much and code should be refactored to make use of helper functions like wait_boot more and strip down the code within installation/bootloader. So that would be the next step. In case you hit any further problems with booting on aarch64 please keep this in mind.